packages/components/src/components/data-list/data-list.jsx

```diff
@@ -1,74 +1,81 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 import { TransitionGroup } from 'react-transition-group';
 import { CellMeasurer, CellMeasurerCache } from 'react-virtualized/dist/es/CellMeasurer';
 import { AutoSizer } from 'react-virtualized/dist/es/AutoSizer';
 import { List } from 'react-virtualized/dist/es/List';
 import { isMobile, isDesktop } from '@deriv/shared';
 import DataListCell from './data-list-cell.jsx';
+import DataListRowWrapper from './data-list-row-wrapper.jsx';
 import ThemedScrollbars from '../themed-scrollbars';
 
-class DataList extends React.PureComponent {
-    state = {
-        scrollTop: 0,
-        isScrolling: false,
-    };
-    items_transition_map = {};
+const DataList = React.memo(props => {
+    const [scrollTop, setScrollTop] = React.useState(0);
+    const [isScrolling, setIsScrolling] = React.useState(false);
+    const [loading, setLoading] = React.useState(true);
+
+    const cache = React.useRef();
+    const list_ref = React.useRef();
+    const items_transition_map = {};
+    const is_dynamic_height = !props.getRowSize;
```

Comment:
`props.getRowSize;()`? :thinking:

Reply:
It's to check if `getRowSize` passed to the component or not.